Cython/Compiler/Symtab.py

```diff
@@ -2267,10 +2267,14 @@ def declare_cfunction(self, name, type, pos,
         if u'inline' in modifiers:
             entry.is_inline_cmethod = True
 
+        entry.vtab_type = entry.type
```

Comment:
From the tests failures, this doesn't seem to be the only case where this needs to be set.

Reply:
Or, the failures seem related to functions with fused types, so maybe they need a `vtab_type` as well.